meinberlin/apps/votes/forms.py

```diff
@@ -4,9 +4,35 @@
 from meinberlin.apps.votes.models import VotingToken
 
 
-class TokenForm(forms.Form):
+class VotingTokenWidget(forms.MultiWidget):
+
+    def __init__(self, attrs=None):
+        widgets = (
+            forms.TextInput(attrs={'size': '4',
+                                   'minlength': '4',
+                                   'maxlength': '4'}),
+            forms.TextInput(attrs={'size': '4',
+                                   'minlength': '4',
+                                   'maxlength': '4'}),
+            forms.TextInput(attrs={'size': '4',
+                                   'minlength': '4',
+                                   'maxlength': '4'}),
+        )
+        super(VotingTokenWidget, self).__init__(widgets, attrs)
+
+    def decompress(self, value):
+        if value:
+            n = 4
```

Comment:
I think you could remove this and simply write 4 in the line below?

Reply:
Yes, I thought about this, and then I wondered if we would ever change it and then it would be more explicit. But for it to be more explicit, it needs to be more of a telling variablename than simply 'n' I guess. I will change it! Thanks:)